src/runner/proxypluginwrappers.cpp

```diff
@@ -27,7 +27,7 @@ namespace boost
 using namespace MOBase;
 
 
-#define COMMON_I_PLUGIN_WRAPPER_DEFINITIONS(class_name) \
+#define COMMON_I_PLUGIN_WRAPPER_DEFINITIONS_(class_name, include_requirements) \

```

Comment:
Maybe give this a slightly clearer name. At a glance, it's not obviously a different macro.